pom.xml

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.424</version>
+    <version>1.596.1</version>
```

Comment:
This was the version being used by previous attempts to add this functionality, I just used what they used.  Also moving forward solved the Xstream issues they encountered (xstream 2.6).  Plus - should should plugins not move forward with the code base?